src/NuGet.Core/NuGet.CommandLine.XPlat/PackCommand.cs

```diff
@@ -0,0 +1,145 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using Microsoft.Dnx.Runtime.Common.CommandLine;
+using NuGet.Commands;
+using NuGet.Logging;
+
+namespace NuGet.CommandLine.XPlat
+{
+    class PackCommand : Command
+    {
+        public PackCommand(CommandLineApplication app, Func<ILogger> getLogger)
+        {
+            app.Command("pack", pack =>
+            {
+                pack.Description = Strings.PackCommand_Description;
+
+                var basePath = pack.Option(
+                    "-b|--basePath <basePath>",
```

Comment:
I vote for keeping the flags the same between platforms. Less confusion for people who use both. So, pascal casing and no hyphens.

---
In reply to: [56573367](https://github.com/NuGet/NuGet.Client/pull/392#discussion_r56573367) [](ancestors = 56573367)